src/widgets/requirements/ui/requirements.module.scss

```diff
@@ -0,0 +1,54 @@
+.content {
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+
+  width: 100%;
+  max-width: $content-width;
+  padding: 40px 120px;
+
+  @include media-laptop {
+    padding: 40px 40px;
+  }
+
+  @include media-tablet {
+    padding: 40px 16px;
+  }
+}
+
+.title {
+  margin-block-start: unset;
+  margin-block-end: unset;
+  margin-inline-start: unset;
+  margin-inline-end: unset;
+
+  font-size: 28px;
+  font-weight: $font-weight-regular;
+  line-height: 36px;
+  color: $color-gray-600;
+
+  @include media-laptop {
+    font-size: 22px;
+  }
+}
```

Comment:
Once you start using the Subtitle component, this set of styles will become unnecessary in whole or in part

Reply:
Done